.github/workflows/ci-build.yml

```diff
@@ -16,6 +16,8 @@ jobs:
     - uses: actions/checkout@v2
     - name: Lint
       run: cargo fmt --message-format human -- --check
+    - name: Clippy
+      run: cargo clippy --all-targets -- -D warnings
```

Comment:
We could use both `--workspace` and `--all-targets` for completeness.

Reply:
Hm, that's a good point -- this does seem to check "more" than with either flag on its own.

We should probably do this in other repos as well, I'll but together a PR for Janus shortly.